src/pages/general/notFound/components/body404.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+
+export const Body404 = () => {
```

Comment:
We are following this convention:

 - Filename: we can just call it _body.tsx_ the reason behind this is that the suffix can be already infered from the path (notFound/components/body.tsx).

- Function name: here we specify the full name (to avoid collisions with other functions, in this case is not likely to happen), in this case the page is called _PageNotFound_ this comopnent should be called _BodyNotFound_ just to keep consistency. There is an additional issue (fault from my side sorry), we are separating folders composed names using a dash.

Summarizing.

**Filenames:**

_src/pages/general/not-found/page.tsx_
_src/pages/general/not-found/components/body.tsx_
_src/pages/general/not-found/components/header.tsx_

Usage

_src/pages/general/not-found/page.tsx_

```javascript
import * as React from 'react';
import {NotFoundHeader} from './components/header';
import {NotFoundBody} from './components/body';



export const NotFoundPage = () => {
       return (
         <div className="row pageError404">
           <div className="col-md-8 col-md-offset-2">
             <div className="panel panel-danger">
               <NotFoundHeader />
               <NotFoundBody />
             </div>
           </div>
         </div>
       );
}
```



_src/pages/general/not-found/components/header.tsx_

```javascript
```